.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,5 @@
+### What would you like to change?
```

Comment:
we don't normally include these files any more they are all in https://github.com/hmcts/.github

that way we can update centrally if we want to and teams are welcome to add their own

Reply:
sure, just copied over from the other one :)